extensions-core/hdfs-storage/pom.xml

```diff
@@ -377,6 +377,11 @@
           <artifactId>junit</artifactId>
           <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
`protobuf-java` (required while running `HdfsFirehoseFactoryTest`) was initially coming as a transitive dependency through `druid-server` -> `avatica-core`
But since `avatica-core` has been removed from `druid-server` in this PR, `protobuf-java` has been explicitly added here.